draft-ietf-rats-corim.md

```diff
@@ -70,6 +70,7 @@ contributor:
 normative:
   RFC4122: uuid
   RFC5280: pkix-cert
+  RFC7250: raw-pk
```

Comment:
This RFC doesn't define raw public key directly. Rather it defines: 
```
case RawPublicKey:
              opaque ASN.1_subjectPublicKeyInfo<1..2^24-1>;
```
But this seems to be referring to a TLS encoding of ASN.1 that is defined in RFC5280.

The key thumbprint refers to a digest of a raw key (as defined by `tagged-pkix-base64-key-type`) 
https://datatracker.ietf.org/doc/html/rfc7093 describes techniques for generating digests of SubjectPublicKeyInfo that is DER encoding. Since another PR added DER encoded key type. Possibly, it makes sense to define a thumbprint type that is the digest of the DER encoded SubjectPublicKeyInfo?

I'm not aware of an RFC that defines a thumbprint for a PEM encoded SubjectPublicKeyInfo, but it may be sufficient for say that any digest algorithm can be applied to the PEM key. There isn't a need to reference anything as the output of a hash algorithm is bytes (bstr).

Reply:
I believe there is a need to describe the algorithm since key fingerprinting is ambiguous and folks will want to get around the rule that there should be no nontrivial key comparisons by converting their keys into multiple formats (also is there a meaningful difference to diverge from the more common term 'fingerprint' with 'thumbprint'?)

Reply:
> I believe there is a need to describe the algorithm since key fingerprinting is ambiguous and folks will want to get around the rule that there should be no nontrivial key comparisons by converting their keys into multiple formats

Method 1 described in [S4.2.1.2 of RFC5280](https://datatracker.ietf.org/doc/html/rfc5280#section-4.2.1.2)?

> (also is there a meaningful difference to diverge from the more common term 'fingerprint' with 'thumbprint'?)

Maybe we could just call it "key identifier" and avoid any colloquialisms.